src/Resources/views/includes/layout/_content.html.twig

```diff
@@ -12,7 +12,7 @@
         {{ attr|map((value, attr) => "#{attr}=\"#{value}\"")|join(' ')|raw }}
     >
         <dt class="whatwedo-utility-bold text-neutral-500 overflow-hidden truncate" id="wwd-crud-block-{{ block.acronym }}-content-{{ contentAcronym }}-label" title="{{ content.option('label')|trans }}">
```

Comment:
@rugbymauri for the title attribute we should strip all tags, as it would look wierd in the popup.